README.md

```diff
@@ -194,38 +194,44 @@ Running this code as-is should generate a structure as shown below:
 ```
 example-organization/
 └── fldr-development
-    ├── prj-bu1-d-env-kms
-    ├── prj-bu1-d-sample-floating
-    ├── prj-bu1-d-sample-base
-    ├── prj-bu1-d-sample-restrict
-    ├── prj-bu1-d-sample-peering
-    ├── prj-bu2-d-env-kms
-    ├── prj-bu2-d-sample-floating
-    ├── prj-bu2-d-sample-base
-    ├── prj-bu2-d-sample-restrict
-    └── prj-bu2-d-sample-peering
+    └── fldr-development-bu1
+        ├── prj-bu1-d-env-kms
+        ├── prj-bu1-d-sample-floating
+        ├── prj-bu1-d-sample-base
+        ├── prj-bu1-d-sample-restrict
+        ├── prj-bu1-d-sample-peering
+    └── fldr-development-bu2
+        ├── prj-bu2-d-env-kms
+        ├── prj-bu2-d-sample-floating
+        ├── prj-bu2-d-sample-base
+        ├── prj-bu2-d-sample-restrict
+        └── prj-bu2-d-sample-peering
 └── fldr-non-production
```

Comment:
For all instances of "non-production" in resource ids, remove the hyphen like "nonproduction".

Context: Other environment names don't use a hyphen, using a hyphen for non-production makes the number of hyphens across environments inconsistent, so it's difficult to script over environments using regexp etc.